experimenter/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -22,7 +26,31 @@ def handle(self, *args, **options):
             NimbusFeatureConfig.objects.values_list("application", "slug")
         )
 
-        for feature in Features.all():
+        # A mapping of (application, slug) to feature configs.
+        #
+        # Features will include multiples of the same feature, each with a
+        # different version. We only need to create a single NimbusFeatureConfig
+        # for each of those, so if we keep track of them we can skip a bunch of
+        # update_or_create() calls.
+        feature_configs: dict[tuple[str, str], NimbusFeatureConfig] = {}
+
+        # Iterate through the unversioned features first for the initial
+        # update_or_create() calls. The unversioned features are the most
+        # up-to-date (since they come from the main branch (or equivalent)), so
+        # lets use the descriptions from there.
```

Comment:
Mmmm interesting.  I wonder if we want to make some kind of wrapper around a feature that lets us manage which fields come from where, ie descriptions from latest main, a main schema, or versioned schemas, etc?  So we can just do that kind of lookup in one place and then everywhere we consume them we don't have to reason about looking through unversioned/versioned to find some particular field.  Really depends on where/how we consume those fields.  Just musing.

Reply:
Yeah i think in the future we want to move the description into the `NimbusVersionedSchema` because we could end up in a case where two features with the same name N versions apart aren't actually the same feature any more.

I'm not sure of a good way to present all this to the user right now, though, so I left the description on the `NimbusFeatureConfig` model, since that is what we are using to generate the UI on the branch page. Moving it into `NimbusVersionedSchema` would require a large refactoring of the front-end to support that.

Reply:
Totally, I mean you don't need to do any of that refactoring really, just change description to a property that manages that lookup.  This is a general pattern we've used in lots of places, provide a facade of a field to the APIs/frontend that actually reasons about doing lookups, similar pattern to how we manage the dates.